nservicebus/pipeline/manipulate-with-behaviors_registernew_core_[6.0,).partial.md

```diff
@@ -5,4 +5,6 @@ snippet:RegisterBehaviorEndpointConfiguration
 
 Behaviors can also be registered from a `Feature` as shown below:
 
-snippet:RegisterBehaviorFromFeature
\ No newline at end of file
+snippet:RegisterBehaviorFromFeature
+
+WARNING: Behaviors are only created once and the same instance is reused on every invocation of the pipeline, even when registering the behavior to be created by a Dependency Injection Container. Never use dependency injection for dependencies configured with an *InstancePerUnitOfWork* scope.
```

Comment:
IMO it's still not clear to understand. How about:

WARNING: Behaviors are only created once and the same instance is reused on every invocation of the pipeline. Consequently, every behaviour dependency will also behave as a singleton, even if a different option was specified when registering it in the DI container.

I'm still not sure why you say this though:
>  Never use dependency injection for dependencies configured with an *InstancePerUnitOfWork* scope.
 
What should they use in such situation?